spec/thor/zsh_completion/generator_spec.rb

```diff
@@ -8,6 +8,13 @@ def baz
         p options
         puts "baz"
       end
+      map 'b' => 'baz'
+
+      desc "foo-bar", "Dashed command"
```

Comment:
Oh, you're right. I was having problem with completion, and I thought dashes cannot be part of a function name. I'll update PR. Thanks for reviewing.